app/models/subjects/ucas_subject_to_dfe_subject_mappings.rb

```diff
@@ -0,0 +1,21 @@
+module Subjects
+  class UCASSubjectToDFESubjectMappings
```

Comment:
I had to go through and `UCASSubjectToDFESubjectMappings` carefully to see how these are different. :) What about `UCASSubjectToDFESubjectMappingCollection`? Oof, it is a bit long ...

Reply:
Thanks, fixed in 0e4eec2